simuleval/data/segments.py

```diff
@@ -15,6 +15,7 @@ class Segment:
     finished: bool = False
     is_empty: bool = False
     data_type: str = None
+    tgt_lang: str = ""
```

Comment:
I see you have made some changes to add target langauge as a property to dataloader and segment classes. But I don't see any change for passing the target language parameter from the dataloader to the segment.

In case you are wondering, the instance class (eg: https://github.com/facebookresearch/SimulEval/blob/main/simuleval/evaluator/instance.py) is the one that loads a specific instance/sample  from the dataloader and then creates segments for that instance.